RevenueCat/Scripts/ProductCategory.cs

```diff
@@ -0,0 +1,14 @@
+public partial class Purchases
+{
+    public enum ProductCategory
```

Comment:
Same for this, we will need new methods in `Purchases.cs` to use these instead of the current versions?

Reply:
This is a new property in `StoreProduct` returned by the API, we don't need to pass it to any of the methods in Purchases.cs

Reply:
Ohh you're right... I thought we were using `ProductType`, but seems like we aren't in unity, so we don't need to deprecate it.